client/src/components/map-page/SinglePark.js

```diff
@@ -1,28 +1,80 @@
 import Navigation from "../Navigation";
-import * as parkData from "../../parks.json";
-
+import parksData from "../../parks.json";
 import "./SinglePark.css";
-const parks = parkData.park;
+import { useEffect, useState } from "react";
+import {
+  saveJSONToLocalStorage,
+  getItemsFromLocalStorage,
+  addNotetoLocalStorage,
+} from "../../utils/itemStorage";
+
+function initialLoad() {
+  const retrieveArray = getItemsFromLocalStorage("parkData");
+  if (retrieveArray.length === 0) {
+    return parksData;
+  } else {
+    return retrieveArray;
+  }
+}
+
 export default function SinglePark() {
-  const pathname = window.location.pathname;
+  const [note, setNote] = useState("");
+  const [parkData, setParkData] = useState(initialLoad());
+  const pathname = window.location.pathname; // turns this into useLocation later
   const geo = pathname.slice([13]).replace(/-/g, ".");
+  const selectedPark = parkData.find((park) =>
+    park.coordinates[0].includes(geo)
+  );
+
+  function handleSubmit(e) {
```

Comment:
right now you can add empty notes. You should validate that the user has written something before allowing to submit an empty note